stable/coredns/Chart.yaml

```diff
@@ -1,7 +1,6 @@
 name: coredns
-version: 0.7.0
-description: CoreDNS is a DNS server that chains middleware and provides Kubernetes
-  DNS Services
+version: 1.0.0
```

Comment:
Note that there's `version` and `appVersion`. The latter is still missing. Pleae add it.

Reply:
Done. assuming Version is the Chart's version: bumbed that to 0.8.0 (instead of 1.0.0).